--- DBI-1.601/Makefile.PL~	2007-09-28 17:11:08.000000000 +0100
+++ DBI-1.601/Makefile.PL	2007-10-27 16:55:35.349648759 +0100
@@ -309,19 +309,6 @@
 roadmap_pm = '  . File::Spec->catfile($self->{INST_LIB}, 'DBI', 'Roadmap.pm') . '
 '.q{
 
-config :: $(changes_pm) $(roadmap_pm)
-	$(NOECHO) $(NOOP)
-
-$(changes_pm): Changes
-	$(MKPATH) $(inst_libdbi)
-	$(RM_F) $(changes_pm)
-	$(CP) Changes $(changes_pm)
-
-$(roadmap_pm): Roadmap.pod
-	$(MKPATH) $(inst_libdbi)
-	$(RM_F) $(roadmap_pm)
-	$(CP) Roadmap.pod $(roadmap_pm)
-
 faq:
 	: checkin any local changes not already checked in before overwriting
 	svn commit --message "dbi.tiddlyspot.com FAQ update" dbi.tiddlyspot.com.html
