diff -burN DBI-1.44.orig/Makefile.PL DBI-1.44/Makefile.PL
--- DBI-1.44.orig/Makefile.PL	2004-10-17 23:42:06.423617752 +0200
+++ DBI-1.44/Makefile.PL	2004-10-17 23:42:55.603141328 +0200
@@ -290,8 +290,8 @@
 # make Changes file available as installed pod docs "perldoc DBI::Changes"
 # make Roadmap.pod file available as installed pod docs "perldoc DBI::Roadmap"
 inst_libdbi = ' . File::Spec->catdir($self->{INST_LIB}, 'DBI') . '
-changes_pm = '  . File::Spec->catfile($self->{INST_LIB}, 'DBI', 'Changes.pm') . '
-roadmap_pm = '  . File::Spec->catfile($self->{INST_LIB}, 'DBI', 'Roadmap.pm') . '
+changes_pm = '  . File::Spec->catfile($self->{INST_LIB}, 'DBI', 'Changes.pod') . '
+roadmap_pm = '  . File::Spec->catfile($self->{INST_LIB}, 'DBI', 'Roadmap.pod') . '
 
 config :: $(changes_pm) $(roadmap_pm)
 	$(NOECHO) $(NOOP)
