diff -ur lib/DBI/FAQ.pm lib/DBI/FAQ.pm
--- lib/DBI/FAQ.pm	Wed Mar 28 16:52:31 2001
+++ lib/DBI/FAQ.pm	Wed Apr  4 19:25:57 2001
@@ -1000,3 +1000,6 @@
 <!--#include virtual="/technology/perl/DBI/templatebottom.html" -->
 </BODY>
 </HTML>
+
+=cut
+
diff -ur lib/DBI/Format.pm lib/DBI/Format.pm
--- lib/DBI/Format.pm	Fri Mar 30 16:35:49 2001
+++ lib/DBI/Format.pm	Wed Apr  4 19:26:23 2001
@@ -467,3 +467,5 @@
 
 L<DBI::Shell(3)>, L<DBI(3)>, L<dbish(1)>
 
+=cut
+
diff -ur lib/DBI/ProxyServer.pm lib/DBI/ProxyServer.pm
--- lib/DBI/ProxyServer.pm	Fri Mar 30 14:56:14 2001
+++ lib/DBI/ProxyServer.pm	Wed Apr  4 19:26:57 2001
@@ -586,3 +586,6 @@
 L<dbiproxy(1)>, L<DBD::Proxy(3)>, L<DBI(3)>, L<RPC::PlServer(3)>,
 L<RPC::PlClient(3)>, L<Net::Daemon(3)>, L<Net::Daemon::Log(3)>,
 L<Sys::Syslog(3)>, L<Win32::EventLog(3)>, L<syslog(2)>
+
+=cut
+
